modules/osdk-hh-create-go-api.adoc

```diff
@@ -0,0 +1,36 @@
+// Module included in the following assemblies:
+//
+// * operators/operator_sdk/helm/osdk-hybrid-helm.adoc
+
+:_content-type: PROCEDURE
+[id="osdk-hh-create-go-api_{context}"]
+= Creating a Go API
+
+Use the Operator SDK CLI to create a Go API.
+
+.Procedure
+
+. Run the following command to create a new custom resource definition (CRD) API with group `cache`, version `v1`, and kind `MemcachedBackup`:
++
+[source,terminal]
+----
+$ operator-sdk create api \
+    --group=cache \
+    --version v1 \
+    --kind MemcachedBackup \
+    --resource \
+    --controller \
+    --plugins=go/v3
+----
+
+. When prompted, enter `y` for creating both resource and controller:
++
+[source,terminal]
+----
+$ Create Resource [y/n]
+y
+Create Controller [y/n]
+y
+----
+
+This procedure generates the `MemcachedBackup` resource API at `api/v1/memcachedbackup_types.go` and the controller at `controllers/memcachedbackup_controller.go`.
```

Comment:
You point out where the new API is created. Should you do the same for the Helm API?